db/index.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const URI = 'mongodb://localhost/listings';
```

Comment:
Should you maybe include the port number?

- "mongodb://localhost:27017/test" 

I think I remember, Matt, mentioning that we might want to include it.

Reply:
Good idea, let me add that. Thanks!